code/development-env/docker/README.md

```diff
@@ -1,5 +1,24 @@
 # Setup local development environment using docker-compose
 
+## Set-up minikube
+
+- Start minikube
+- Enable ingress controller
+  - `minikube addons enable ingress`
+- Create namespace
+  - `kubectl create namespace devtest`
+- Change default namespace
+  - `kubectl config set-context minikube --namespace=devtest`
```

Comment:
Is it worth adding a note here to suggest installation of `kubectx` and `kubens`? - https://github.com/ahmetb/kubectx